charts/osm/templates/osm-deployment.yaml

```diff
@@ -6,20 +6,22 @@ metadata:
     app: osm-controller
     meshName: {{ .Values.OpenServiceMesh.meshName }}
 spec:
-  replicas: {{ .Values.OpenServiceMesh.replicaCount }}
+  replicas: 1
```

Comment:
Right now we can only support one controller pod and hence this isn't configurable